pkgs/applications/audio/gwc/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv
+, fetchFromGitHub
+, autoreconfHook
+, pkg-config
+, alsaLib
+, libpulseaudio
+, gtk2
+, hicolor-icon-theme
+, libsndfile
+, fftw
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gwc";
+  version = "0.22-04";
+
+  src = fetchFromGitHub {
+    owner = "AlisterH";
+    repo = pname;
+    rev = version;
+    sha256 = "0xvfra32dchnnyf9kj5s5xmqhln8jdrc9f0040hjr2dsb58y206p";
+  };
+
+  buildInputs = [
```

Comment:
We prefer having `nativeBuildInputs` before `buildInputs`, see https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887.

Reply:
Thanks, all done.